pkg/daemonset/daemonset.go

```diff
@@ -138,6 +138,71 @@ func (builder *Builder) WithNodeSelector(selector map[string]string) *Builder {
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to pod's definition.
```

Comment:
It's not pod definition. It's daemonset definition.

Reply:
updated as suggested